packages/plugins/rubygem-foreman_supervisory_authority/rubygem-foreman_supervisory_authority.spec

```diff
@@ -1,34 +1,25 @@
-# Generated from foreman_supervisory_authority-0.0.1.gem by gem2rpm -*- rpm-spec -*-
 # template: foreman_plugin
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
 %global gem_name foreman_supervisory_authority
 %global plugin_name supervisory_authority
-%global foreman_min_version 1.20
+%global foreman_min_version FIXME
```

Comment:
Looks like something went wrong here and I'm guessing https://github.com/theforeman/foreman-packaging/pull/11860 was opened to fix it.